### DISCUSSION TITLE
Can't find schedulers/runs

### DISCUSSION THREAD

**Discussion by enpro-github** [NodePit/runner#1] [Q&A]
I randomly started getting this error this morning, where I can't views the schedules or runs:

![image](https://github.com/NodePit/runner/assets/68210951/562ed6b8-b812-48ff-8885-29a0b790d8f5)

![image](https://github.com/NodePit/runner/assets/68210951/d3f49a09-4d95-49d3-aa0b-357f264dcdbc)

![image](https://github.com/NodePit/runner/assets/68210951/34d1a888-864e-40e4-8ecd-c95e3643ddff)

I tried rebuilding the Docker image but to no avail, what could cause this? Everything else works fine.

Thank you

**Comment by qqilihq** [comment]
In touch via email - closing.
